app/views/layouts/application.html.erb

```diff
@@ -36,6 +36,7 @@
     )%>);
 
     new ELMO.Views.MissionChangeDropdown({el: '#change_mission select'});
+    new ELMO.Views.RegenerableFieldView();
```

Comment:
I actually had a question about that buried in one of my comments:

> Let me know if app/views/layouts/application.html.erb was the right place to include the Backbone view, since I wasn't sure whether to somehow try to include it directly from f.regenerable_field (e.g. by adding a `<script>` tag in the output of that method).

The alternative would be that a developer would add `f.regenerable_field` to a page and have to remember to add `new ELMO.Views.RegenerableFieldView()` themselves, which seems like a recipe for frustration.